alsongDalsong/ASMusicKit/ASMusicKit/ASMusicAPI.swift

```diff
@@ -90,7 +86,7 @@ public struct ASMusicAPI {
                     throw ASMusicErrors(type: .search, reason: "", file: #file, line: #line)
```

Comment:
민하님께서 작업하신 부분은 아닌 것 같지만 여기도 조금 더 상세하게 에러를 던져주면 어떨까 싶습니다!

```suggestion
                    throw ASMusicErrors(type: .search, reason: error.localizedDescription, file: #file, line: #line)
```